src/tests/ftest/soak/soak_utils.py

```diff
@@ -59,10 +58,8 @@ def add_pools(self, pool_names):
     for pool_name in pool_names:
         path = "".join(["/run/", pool_name, "/*"])
         # Create a pool and add it to the overall list of pools
-        self.pool.append(TestPool(self.context, self.dmg_command))
-        self.pool[-1].namespace = path
-        self.pool[-1].get_params(self)
-        self.pool[-1].create()
+        self.pool.append(self.get_pool(namespace=path, connect=False))
+        self.pool[-1].set_property("reclaim", "time")
```

Comment:
Should not need to do a set_property.  I had created a basic property to allow setting properties from yaml 

This needs to be removed because it will break soak from setting ec_cell_sz property from yaml

Reply:
Done